cli/src/main/java/io/kestra/cli/commands/namespaces/files/NamespaceFilesUpdateCommand.java

```diff
@@ -0,0 +1,77 @@
+package io.kestra.cli.commands.namespaces.files;
+
+import io.kestra.cli.AbstractValidateCommand;
+import io.kestra.cli.commands.AbstractServiceNamespaceUpdateCommand;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.client.exceptions.HttpClientResponseException;
+import io.micronaut.http.client.multipart.MultipartBody;
+import io.micronaut.http.client.netty.DefaultHttpClient;
+import lombok.extern.slf4j.Slf4j;
+import nl.basjes.gitignore.GitIgnore;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+
+@CommandLine.Command(
+    name = "update",
+    description = "update namespace files",
+    mixinStandardHelpOptions = true
+)
+@Slf4j
+public class NamespaceFilesUpdateCommand extends AbstractServiceNamespaceUpdateCommand {
+
+    private static final String KESTRA_IGNORE_FILE = ".kestraignore";
+
+    @Override
+    public Integer call() throws Exception {
+        super.call();
+
+        try (var files = Files.walk(directory); DefaultHttpClient client = client()) {
+            if (delete) {
+                client.toBlocking().exchange(this.requestOptions(HttpRequest.DELETE(apiUri("/files/namespaces/") + namespace + "?path=/", null)));
+            }
+
+            GitIgnore gitIgnore = parseKestraIgnore(directory);
+
+            List<Path> paths = files
+                .filter(Files::isRegularFile)
+                .filter(path -> !path.endsWith(KESTRA_IGNORE_FILE))
+                .filter(path -> !Optional.ofNullable(gitIgnore.isIgnoredFile(path.toString())).orElse(false))
```

Comment:
This is a bit hard to read, I check the code and they uses Boolean as a 3-way boolean :(

Maybe `!Boolean.TRUE.equals(gitIgnore.isIgnoredFile(path.toString()))` would be more readable?